AWSCognitoAuth/AWSCognitoAuth.m

```diff
@@ -216,7 +216,40 @@ - (BOOL) isSignedIn {
     return refreshToken!=nil;
 }
 
-#pragma mark sign in
+- (void)launchSignInWithViewController:(UIViewController *) vc
+            completion:(nullable AWSCognitoAuthGetSessionBlock) completion {
+
+    __block __weak NSOperation *weakGetSessionOperation;
+    NSOperation *getSessionOperation = [NSBlockOperation blockOperationWithBlock:^{
+        self.presentationAnchor = nil;
```

Comment:
Could refactor this entire block to be reused b/t `launchSignInWithViewController` and `launchSignInWithWebUI`